spring-cloud-starter-task-composedtaskrunner/src/test/java/org/springframework/cloud/task/app/composedtaskrunner/ComposedTaskRunnerStepFactoryTests.java

```diff
@@ -82,7 +104,7 @@ public DataSource getTaskDataSource() {
 		}
 
 		@Bean
-		public ComposedTaskRunnerStepFactory stepFactory() {
+		public ComposedTaskRunnerStepFactory stepFactory(TaskProperties taskProperties) {
```

Comment:
Why is this being passed in if it is not used.

Reply:
Updated test so that the requirement for TaskProperty can be clearly seen.